src/main/java/org/prebid/server/auction/AuctionRequestFactory.java

```diff
@@ -128,7 +128,7 @@ private ObjectNode populateBidRequestExtension(ObjectNode ext) {
                     prebid.getAliases(),
                     prebid.getBidadjustmentfactors(),
                     ExtRequestTargeting.of(populatePriceGranularity(targeting.getPricegranularity()),
-                            targeting.getIncludewinners()),
+                            targeting.getIncludewinners() != null ? targeting.getIncludewinners() : true),
```

Comment:
It was my mistake to create new ExtRequestTargeting each time targeting != null, can I ask you to update populateBidRequestExtension and populatePriceGranularity with code below:


  ```
 private ObjectNode populateBidRequestExtension(ObjectNode ext) {
        final ExtBidRequest extBidRequest;
        try {
            extBidRequest = Json.mapper.treeToValue(ext, ExtBidRequest.class);
        } catch (JsonProcessingException e) {
            throw new InvalidRequestException(String.format("Error decoding bidRequest.ext: %s", e.getMessage()));
        }
        final ExtRequestPrebid prebid = extBidRequest.getPrebid();
        final ExtRequestTargeting targeting = prebid != null ? prebid.getTargeting() : null;

        if (targeting == null) {
            return null;
        }

        final boolean isPriceGranularityNull = targeting.getPricegranularity().isNull();
        final boolean isPriceGranularityTextual = targeting.getPricegranularity().isTextual();
        final Boolean updatedWinners = targeting.getIncludewinners() == null;

        if (updatedWinners || isPriceGranularityNull || isPriceGranularityTextual) {
            return Json.mapper.valueToTree(ExtBidRequest.of(ExtRequestPrebid.of(
                    prebid.getAliases(),
                    prebid.getBidadjustmentfactors(),
                    ExtRequestTargeting.of(
                            populatePriceGranularity(targeting.getPricegranularity(), isPriceGranularityNull,
                                    isPriceGranularityTextual),
                           updatedWinners ? targeting.getIncludewinners() : true),
                    prebid.getStoredrequest(),
                    prebid.getCache())));
        }
        return null;
    }

    /**
     * Populates priceGranularity with converted value.
     * <p>
     * In case of valid string price granularity replaced it with appropriate custom view.
     * In case of invalid string value throws {@link InvalidRequestException}.
     * In case of missing Json node sets default custom value.
     */
    private JsonNode populatePriceGranularity(JsonNode priceGranularityNode, boolean isPriceGranularityNull,
                                              boolean isPriceGranularityTextual) {
        if (isPriceGranularityNull) {
            return Json.mapper.valueToTree(PriceGranularity.DEFAULT.getBuckets());
        }
        if (isPriceGranularityTextual) {
            final PriceGranularity priceGranularity;
            try {
                priceGranularity = PriceGranularity.createFromString(priceGranularityNode.textValue());
            } catch (PreBidException ex) {
                throw new InvalidRequestException(ex.getMessage());
            }
            return Json.mapper.valueToTree(priceGranularity.getBuckets());
        }
        return priceGranularityNode;
    }
```